test/fuzz/CMakeLists.txt

```diff
@@ -3,6 +3,15 @@ include_directories(${PROJECT_SOURCE_DIR}/src)
 include_directories(${PROJECT_BINARY_DIR}/src)
```

Comment:
Is it really worth making two separate commits? I would rather squash them.

Reply:
I have no strict preference here. I'll do.

Reply:
I did.